src/e2e-test/java/io/cdap/plugin/bigquery/stepsdesign/BQValidation.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.bigquery.stepsdesign;
+
+import com.google.cloud.Timestamp;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.Field;
+import com.google.cloud.bigquery.Schema;
+import com.google.cloud.bigquery.TableId;
+import com.google.cloud.bigquery.TableResult;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import io.cdap.e2e.utils.BigQueryClient;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import org.apache.spark.sql.types.Decimal;
+import org.joda.time.DateTime;
+import org.junit.Assert;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.sql.Time;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+/**
+ * BigQuery Plugin validation.
+ */
+public class BQValidation {
+
+  public static void main(String[] args) throws IOException, InterruptedException {
+    validateSourceBQToTargetBQRecord("E2E_SOURCE_32bee2c2_8e92_46ab_86af_9f9333371f58",
+                                     "E2E_TARGET_9905c77f_c04d_4958_accc_de3f02de1bba");
```

Comment:
what are these hardcoded UUIDs??

Reply:
The main function for fixing e2e build failures with source and sink tables was added on local. Now that they've passed. I've now removed it.